### ISSUE TITLE
Support embedded PDB

### ISSUE THREAD

**Issue by SIkebe** [libgit2/libgit2sharp#1857]
In .NET Core 3.X, there is a [workaround for single file exe with libgit2sharp](https://github.com/libgit2/libgit2sharp/issues/1754).

However, in .NET 5.0, there is no support for `IncludeSymbolsInSingleFile` and Microsoft recommends embedded PDBs instead.
https://github.com/dotnet/runtime/issues/42278#issuecomment-693036257

Could you consider using embedded PDB?

```cmd
mkdir Sample && cd Sample
dotnet new console
dotnet add package libgit2sharp

# Modify csproj file (See below)

dotnet publish -c Release -r win-x64 --self-contained=true /p:PublishSingleFile=true /p:IncludeNativeLibrariesForSelfExtract=true
```

```csharp
using System;

namespace Sample
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }
}
```

Sample.csproj
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
    <DebugType>embedded</DebugType>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="libgit2sharp" Version="0.26.2" />
  </ItemGroup>

</Project>
```
![image](https://user-images.githubusercontent.com/17608272/101356514-8678f780-38db-11eb-917b-65a0d88fca98.png)

**Comment by CyberSinh** [comment]
I don't know what workaround could be used, but it would be more useful to deal with this problem at the root and stop making a PDB file mandatory to start an application. A ticket (https://github.com/libgit2/libgit2sharp.nativebinaries/issues/111) has been posted for this purpose but no answer so far.

**Comment by bording** [comment]
This has been fixed with 0.27.0-preview-0116.

**Comment by SIkebe** [comment]
I've confirmed this issue solved. Thanks!
